src/scripting/lua_audio.cpp

```diff
@@ -272,10 +269,6 @@ static int impl_track_get(lua_State* L) {
 		return luaL_error(L, "Error: Attempted to access an invalid music track.\n");
 	}
 	const char* m = luaL_checkstring(L, 2);
-	return_bool_attrib("valid", track->valid());
```

Comment:
@CelticMinstrel is this alright to remove, or should it always return `true` and a deprecation message added?